RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate LanguageView to TypeScript

## Proposed changes
The proposal of this PR is migrate `LanguageView` to TypeScript.


## Issue(s)	
N/A

## How to test or reproduce
On the app, open the menu, go to settings -> Language. Then check if you can change the language app normally.

## Screenshots
N/A

## Types of changes




## Checklist



## Further comments
N/A
